packages/block-library/src/media-text/index.js

```diff
@@ -20,6 +20,7 @@ export { metadata, name };
 export const settings = {
 	icon,
 	example: {
+		viewportWidth: 601, // Columns collapse "@media (max-width: 600px)".
```

Comment:
I think if we want we can avoid adding a new viewPortWidth setting by using isStackedOnMobile false attributes on columns and media-text blocks.

Reply:
That could work, yes.

Reply:
I'm not sure if that's the best approach though. Why do we set previews to such a small width by default?

Reply:
Good question, I'm not sure why the previews are so small by default. I guess it may make sense to explore changing the default.